examples/cockroachdb/cockroachdb-blueprint.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: cockroachdb-blueprint
+  namespace: kanister
+actions:
+  backup:
+    outputArtifacts:
+      cockroachDBCloudDump:
+        keyValue:
+          s3path: '/cockroachdb-backups/{{ .StatefulSet.Namespace }}/{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}'
+    phases:
+      - func: KubeExec
+        name: backupToS3
+        args:
+          namespace: "{{ .StatefulSet.Namespace }}"
+          pod: "cockroachdb-client-secure"
```

Comment:
1. Is this the correct pod name? Is this not a pod created by StatefulSet?
2. Instead of hardcoding pod name, can we derive it from the StatefulSet object.